filebeat/_meta/config/filebeat.inputs.reference.yml.tmpl

```
@@ -280,7 +280,7 @@ filebeat.inputs:
 
   ### Prospector options
 
-  # How often the input checks for new files in the paths that are specified
+  # How often do the input checks for new files in the paths that are specified
```

```suggestion
  # How often the input checks for new files in the paths that are specified
```
I think the original is correct in this case.